docs/07_admin/02_RESTadmin.rst

```diff
@@ -0,0 +1,897 @@
+.. REST API Guide
+.. https://docs.google.com/document/d/1JJDP7g25SWerNVCxgff02qp9msHbqA9nt3RAFx8-Qng
+
+.. |br| raw:: html
+
+   <br />
+
+.. |ar| raw:: html
+
+   <div align="right">
+
+.. Images
+
+
+.. Links
+
+
+.. =============================================
+
+
+****************************
+Administration API Reference
+****************************
+
+This section presents the list of administrative API methods in alphabetical order.
+
+
+Audit Trail
+===========
+
+The audit trail API is used to interact with the audit trail log tables in the storage microservice. In FogLAMP, log information are stored in the system log where the microservice is hosted. All the relevant information used for auditing are instead stored inside FogLAMP and they are accessible through the Admin REST API. The API allow the reading but also the addition of extra audit logs, as if such logs are created within the system.
```

Comment:
log information **is** stored in the system log

information for auditing **is** instead stored...

The API **allows** the reading...